node/release_template.yml

```diff
@@ -0,0 +1,90 @@
+on:
+  push:
+    branches: [RELEASE_BRANCH]
+  workflow_dispatch: {}
+
+permissions:
+  contents: write
+  pull-requests: write
+  id-token: write
+
+name: release-RELEASE_TAG
```

Comment:
One thing I did expect us to be able to fix eventually was a release workflow per branch. Shouldn't we be able to calculate this based on the context, then `branches:[]` can actually be a list? 

something like ` branch == "main" ? "latest" : $tagFromBranch `

Reply:
I don't understand what you're asking for here.  Are you asking for the tag to be optional and to calculate it if it's not provided in the generation script?

Reply:
I am asking that we have one release workflow for a repository rather than one per release branch.